### ISSUE TITLE
启动时页面不断刷新

### ISSUE THREAD

**Issue by iwxyi** [gitbito/bitoai#9]
Type: <b>Bug</b>

启动vscode的时候，恢复之前打开的文件（有多个）会一直按顺序切换，到最后一个会从第一个重新开始切换，速度很快，导致界面一直在闪动

Extension version: 1.0.130
VS Code version: Code 1.74.1 (1ad8d514439d5077d2b0b7ee64d2ce82a9308e5a, 2022-12-14T10:30:51.966Z)
OS version: Windows_NT x64 10.0.19042
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-12700K (20 x 3610)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|63.81GB (39.01GB free)|
|Process Argv|--crash-reporter-id 6143b345-6d8a-464f-8e76-e0b392c41d8b|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30669237
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848:30662994
nodejswelcome1:30587005
2e4cg342:30602488
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
a9j8j154:30646983
vscodeenable:30660116
pythonb192:30669360

```

</details>

**Comment by bitoandy** [comment]
Hi @iwxyi ,

We have identified the issue and the fix will be available in the next release.
translated via google below:
我们已经确定了该问题，并将在下一版本中提供修复。

**Comment by nisha-bito** [comment]
Hi @iwxyi ,

Please upgrade to the Bito version 1.0.132. This issue is resolved.
In case you still see the issue after upgrade, please report it here.

Thanks,
Nisha

**Comment by nisha-bito** [comment]
hi @iwxyi ,

I hope you are able to use the Bito plugin without any issue now. The issue reported by you was fixed in previous release of Bito. 
We also have released 1.0.134 with some more enhancements. Please do let us know if you see any issue.

Thanks,
Nisha
